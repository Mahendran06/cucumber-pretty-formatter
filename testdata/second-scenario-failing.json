{"event": "SourceRead", "feature_source_id": "features/my.feature:1", "source": "Feature: My feature\n  In order to figure out which events we need to listen to\n  We should be able to pretty print semi-complex feature output\n\n  Scenario: First scenario\n    Given passing\n    And passing\n\n  Scenario: Second scenario\n    Given passing\n    And failing\n    And skipping"}
{"event": "TestCaseStarted", "location": "features/my.feature:5"}
{"event": "TestStepStarted", "location": "features/my.feature:6"}
{"event": "TestStepMatched", "location": "features/my.feature:6", "step_definition_id": "FeatureContext::passing()"}
{"event": "TestStepFinished", "location": "features/my.feature:6", "status": "passed"}
{"event": "TestStepStarted", "location": "features/my.feature:7"}
{"event": "TestStepMatched", "location": "features/my.feature:7", "step_definition_id": "FeatureContext::passing()"}
{"event": "TestStepFinished", "location": "features/my.feature:7", "status": "passed"}
{"event": "ScenarioHasPassed", "location": "features/my.feature:7"}
{"event": "TestCaseStarted", "location": "features/my.feature:9"}
{"event": "TestStepStarted", "location": "features/my.feature:10"}
{"event": "TestStepMatched", "location": "features/my.feature:10", "step_definition_id": "FeatureContext::passing()"}
{"event": "TestStepFinished", "location": "features/my.feature:10", "status": "passed"}
{"event": "TestStepStarted", "location": "features/my.feature:11"}
{"event": "TestStepMatched", "location": "features/my.feature:11", "step_definition_id": "FeatureContext::failing()"}
{"event": "TestStepFinished", "location": "features/my.feature:11", "status": "failed", "error_summary": "Something broke", "error_details": "some\nexplanation"}
{"event": "TestStepStarted", "location": "features/my.feature:12"}
{"event": "TestStepMatched", "location": "features/my.feature:12", "step_definition_id": "FeatureContext::skipping()"}
{"event": "TestStepFinished", "location": "features/my.feature:12", "status": "skipped"}
{"event": "ScenarioHasFailed", "location": "features/my.feature:5"}
{"event": "FinishedTestingFeature", "feature_source_id": "features/my.feature:1"}
{"event": "TestingHasFinished", "time": "0m0.01s", "memory": "6.73Mb"}
