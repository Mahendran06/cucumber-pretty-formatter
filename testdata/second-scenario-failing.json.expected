Feature: My feature
  In order to figure out which events we need to listen to
  We should be able to pretty print semi-complex feature output

  Scenario: First scenario # features/my.feature:5
    Given passing # FeatureContext::passing()
    And passing # FeatureContext::passing()

  Scenario: Second scenario # features/my.feature:9
    Given passing # FeatureContext::passing()
    And failing # FeatureContext::failing()
      Something broke
      some
      explanation
    And skipping # FeatureContext::skipping()

2 scenarios (1 passed, 1 failed)
5 steps (3 passed, 1 failed, 1 skipped)
0m0.01s (6.73Mb)
